site/components/footer/index.js

```diff
@@ -20,31 +21,43 @@ import mapSVG from './SVG/map.svg';
 
 const cx = classnames.bind(styles);
 
+const trackAnalytics = title => () => ReactGA.event({
+  category: 'FooterNavigation',
+  action: title,
+  label: 'click',
+});
```

Comment:
OK - I think its probably not worth doing. But the limitation is worth highlighting to whoever will be looking at analytics.